### DISCUSSION TITLE
Rule for YAML simple array indentation

### DISCUSSION THREAD

**Discussion by claudiu-cristea** [eslint/eslint#15259] [Q&A] [answered]
This YAML is valid:

```yaml
foo:
  - bar
    - baz
      - qux
```
It decodes to a simple, ordered, array  `['bar', 'baz', 'qux']`.

However, is not _pretty_. Is there a rule that enforces the indentation so that each item starts at the same column?

**Comment by nzakas** [comment, marked as answer]
ESLint is a JavaScript linter. It doesn’t have functionality to beautify YAML code.
